ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/deploy_container/scripts/dockerCPU.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/bash
+
+# Usage - Script takes one argument, the number of cpus for the docker container
+# ie bash dockerCPU.sh 4
+# Argument cannot be 0 or greater than the number of CPUs on the machine
+# If /var/log/docker_cpu does not exist, run: echo 0 > /var/log/docker_cpu
```

Comment:
This seems to be done in the playbooks and not this script, so perhaps we should abort if the file is not present as something will have gone wrong (We could default `start_cpu` to zero in that case I guess and chuck out a warning but I feel aborting might be the better option just in case this script is used by someone without it being from the `docker_container` ansible task.